tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2110,7 +2110,8 @@ <h2>Create a distance feature</h2>
       isComputedKeyStr_: function(feature) {
         return feature == inferenceValueStr || feature == inferenceLabelStr
             || feature == inferenceCorrectStr || feature == inferenceErrorStr
-            || feature == inferenceScoreStr || feature == inferenceAbsErrorStr
+            || feature.lastIndexOf(inferenceScoreStr, 0) == 0
```

Comment:
you can remove the ",0" second parameter from lastIndexOf. It is 0 by default.

Reply:
I think that it defaults to +infinity/[length of the string].